Neos.ContentRepository/Classes/Domain/ValueObject/NodeIdentifier.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Domain\ValueObject;
```

Comment:
I'm currently removing these from the code base in the new CR as they are neither necessary nor helpful. I think we can remove them here as well

Reply:
Sure, as soon as your PR is ready and merged we'll either update this one or create a new PR for the core :)